src/Tizen.NUI.Components/Controls/Menu.cs

```diff
@@ -559,12 +568,7 @@ private View GetRootView()
         // If there is not enought space, then menu's size can be also resized.
         private void CalculateMenuPosition()
         {
-            if ((Anchor == null) || (Content == null))
-            {
-                return;
-            }
-
-            if (Items == null)
+            if (Anchor == null)
```

Comment:
To remove null check of `Content` and `Items` is because they are not used in this method, right?

Reply:
Yes, you are right. Moreover, this PR enables that Content and Items are not set.